### DISCUSSION TITLE
Option to Disable Stats for Users

### DISCUSSION THREAD

**Discussion by dnburgess** [diced/zipline#605] [Ideas]
I feel like there should be an option to disable the Stats page.

Also, it would be nice to make this section on the home page only show relevant information for the user and not the whole server:

![image](https://github.com/user-attachments/assets/0ee1adc6-1f82-4962-bd15-8b4304063d71)

The users don't need to know how many users and files are on the server.

Like a "DISABLE_SERVER_STATS_FOR_USERS" environment variable in Docker would be awesome!

**Comment by diced** [comment]
this has been worked on in v4, the option to limit stats to admins as well as disable it entirely have been added

**Comment by dnburgess** [comment]
Current version appears to be 3.7.9. Any ETA on will version 4 be available? Would love to check it out in Docker.
